### ISSUE TITLE
Should truck go under text on main website?

### ISSUE THREAD

**Issue by cbm755** [flatpak/flatpak.github.io#326]
At large font sizes, the teaser text hides behind the truck:

![Screenshot from 2019-03-15 17-59-34](https://user-images.githubusercontent.com/818622/54468666-37c11c80-474c-11e9-9940-b6741dbbfc80.png)

Versus "normal" font size:

![Screenshot from 2019-03-15 17-59-36](https://user-images.githubusercontent.com/818622/54468672-43144800-474c-11e9-87e1-de5a1a41677d.png)

**Comment by jimmac** [comment]
So far I haven't been able to reproduce this. The media-query breakpoint has been specifically added to address this.

![Screenshot from 2019-03-18 14-07-22](https://user-images.githubusercontent.com/14821/54534118-02a30d00-498c-11e9-9782-88fc912b2f74.png)

![Screenshot from 2019-03-18 14-03-59](https://user-images.githubusercontent.com/14821/54534129-06369400-498c-11e9-98c9-c1d19389078a.png)

**Comment by cbm755** [comment]
I cannot reproduce on chromium.

My Firefox (up-to-date) has "zoom text but not images" either configured somewhere (or maybe Firefox is like that by default).

**Comment by cbm755** [comment]
Yeah, `browser.zoom.full` in `about:config` is set to false.  Maybe this is not the default; if so its probably not worth worrying too much about.

Close if you want.

**Comment by cbm755** [comment]
I should've said: with `browser.zoom.full` set to true, I *cannot* reproduce.

**Comment by razzeee** [comment]
I tested in my firefox and it seems to have `browser.zoom.full` by default. So no problem there.

**Comment by razzeee** [comment]
While the truck has been gone for some time, zooming still has issues with the current main page

**Comment by jimmac** [comment]
Indeed, the splash image breaks the layout.
